websocket/commons/api.w

```diff
@@ -0,0 +1,20 @@
+pub struct RouteOptions {
+  routeKey: str;
+}
+
+pub interface IWebSocket extends std.IResource {
+  onConnect(handler: inflight(str): void): void;
+  onDisconnect(handler: inflight(str): void): void;
+  onMessage(handler: inflight(str, str): void): void;
+
+  initialize(): void;
+
+  inflight sendMessage(connectionId: str, message: str);
+
+  inflight url(): str;
+}
+
+pub struct WebSocketProps {
```

Comment:
is this part of the public api?

Reply:
Yes, it's the props to create the WebSocket. If I remove the `pub`, the classes that implement the interface cannot receive the `props` defined in this file.